shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -888,30 +893,47 @@ - (instancetype)initWithTarget:(id)target
 }
 
 - (void)touchesBegan:(NSSet*)touches withEvent:(UIEvent*)event {
-  [_platformViewsController->getFlutterViewController() touchesBegan:touches withEvent:event];
+  FML_DCHECK(_currentTouchPointersCount >= 0);
+  if (_currentTouchPointersCount < 0) {
+    // _currentTouchPointersCount should never be less than 0.
+    // To not breaking production code, we reset the count to 0 if a negative value
+    // is reached.
+    _currentTouchPointersCount = 0;
+  }
```

Comment:
This shouldn't be possible now that you've changed the cancel logic, right?

Reply:
Right, removed